generator/config/stage/setWindowFields.yaml

```diff
@@ -27,10 +27,3 @@ arguments:
         description: |
             Specifies the field(s) to append to the documents in the output returned by the $setWindowFields stage. Each field is set to the result returned by the window operator.
             A field can contain dots to specify embedded document fields and array fields. The semantics for the embedded document dotted notation in the $setWindowFields stage are the same as the $addFields and $set stages.
-    -
-        name: window
```

Comment:
I double-checked the docs after this was removed. It's a little confusing, as the `window` option (and its nested `document` and `range` fields) are documented in the window stage docs, but seem to be part of an output field.

Reply:
It should be documented as `output.$.window`, `output.$.window.documents`...

Reply:
`output` is not modeled at all, since it's an `object` type above. Does this leave us no way to utilize a builder for `window` and its sub-fields?

Reply:
It's modeled using `MongoDB\Builder\Accumulator::window()` function.